pkg/lib/page.scss

```diff
@@ -151,14 +151,12 @@ a.disabled:hover {
   --ct-color-list-active-text: var(--pf-v6-global--palette--blue-500);
   --ct-color-list-active-bg: var(--ct-color-bg);
   --ct-color-list-active-border: var(--ct-color-list-border);
-  --ct-color-list-critical-bg: var(--pf-t--global--color--nonstatus--red--100);
-  --ct-color-list-critical-border: color-mix(in srgb, var(--pf-t--global--color--nonstatus--red--default), white 50%);
```

Comment:
Forced this to show but now looks more brown?

![image](https://github.com/user-attachments/assets/fbc55091-d097-472d-80c0-d77e2b82529b)